src/mono/wasm/runtime/net6-legacy/method-calls.ts

```diff
@@ -296,4 +296,59 @@ export function mono_wasm_invoke_js_blazor(exceptionMessage: Int32Ptr, callInfo:
         exceptionRoot.release();
         return 0;
     }
-}
\ No newline at end of file
+}
+
+export function mono_wasm_change_case_invariant(exceptionMessage: Int32Ptr, src: number, srcLength: number, dst: number, dstLength: number, toUpper: boolean) : void{
+    try{
+        const input = get_uft16_string(src, srcLength);
+        let result = toUpper ? input.toUpperCase() : input.toLowerCase();
```

Comment:
this ternary will work as expected for ints with zero/non-zero values, so it doesn't have to change.

Reply:
However, is toUpperCase/toLowerCase going to actually use the invariant culture? It seems like it would use the current system-local culture. Wouldn't you want to use toLocaleXXXCase with an invariant locale of some sort? Is that not possible?

Reply:
Yes, we have it in `mono_wasm_change_case`. This snippet belongs to `mono_wasm_change_case_invariant` so we're good.

Reply:
I think we want to use a specific locale here, instead of relying on the user's system locale. The invariant culture is meant to be the same no matter where you run the code, so it's suitable for serialization etc.

One option based on CLDR would be "und", but it's unclear to me whether that would match .NET invariant culture behavior close enough. We may want to pick a specific known-available english locale and use that instead.

I'm OK with it being fixed in a follow-up PR, but this code as-is isn't correct.

Reply:
The link you quoted offline to support this theory: https://262.ecma-international.org/5.1/#sec-15.5.4.16